packages/salesforcedx-vscode-soql/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "salesforcedx-vscode-soql",
   "displayName": "SOQL",
   "description": "Provides code-editing features for SOQL",
-  "qna": "https://github.com/forcedotcom/soql-tooling/issues",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
```

Comment:
You might want to still keep this to `soql-tooling` and just remove the script changes below.

Reply:
fixed